region.go

```diff
@@ -124,20 +132,33 @@ var Regions = map[string]Region{
 			"sao04",
 		},
 		SysTypes: []string{"s922", "e980"},
+		VPCZones: []string{"br-sao-1", "br-sao-2", "br-sao-3"},
 	},
 	"tok": {
 		Description: "Tokyo, Japan",
 		VPCRegion:   "jp-tok",
 		COSRegion:   "jp-tok",
 		Zones:       []string{"tok04"},
 		SysTypes:    []string{"s922", "e980"},
+		VPCZones:    []string{"jp-tok-1", "jp-tok-2", "jp-tok-3"},
 	},
-	"us-east": {
-		Description: "Washington DC, USA",
-		VPCRegion:   "us-east",
-		COSRegion:   "us-east",
-		Zones:       []string{"us-east"},
-		SysTypes:    []string{}, // Missing
+	"tor": {
+		Description: "Toronto, Canada",
+		VPCRegion:   "ca-tor",
+		COSRegion:   "ca-tor",
+		Zones:       []string{"tor01"},
+		SysTypes:    []string{"s922", "e980"},
+		VPCZones:    []string{"ca-tor-1", "ca-tor-2", "ca-tor-3"},
+	},
+	"us-south": {
```

Comment:
How about moving this next to dal region block and adding a comment stating why we added it as separate entry

Reply:
I have kept it in same order to maintain the alpha ascending order. Just added a comment.